samples/Identity.ExternalClaims/README.md

```diff
@@ -0,0 +1,18 @@
+AuthSamples.Identity.ExternalClaims
+=================
+
+Sample demonstrating copying over static and dynamic external claims from Google authentication during login:
+
```

Comment:
See https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/google-logins?tabs=aspnetcore2x for basic setup using google logins